qemu/tests/blockdev_mirror_simple.py

```diff
@@ -0,0 +1,57 @@
+import random
+
+from virttest import utils_numeric
+
+from provider.blockdev_mirror_wait import BlockdevMirrorWaitTest
+
+
+class BlockdevMirrorSimpleTest(BlockdevMirrorWaitTest):
+    """
+    Block mirror simple test:
+      granularity, buf-size
+    """
+
+    def __init__(self, test, params, env):
+        self._set_granularity(params)
+        self._set_bufsize(params)
+        super(BlockdevMirrorSimpleTest, self).__init__(test, params, env)
+
+    def _set_granularity(self, params):
+        granularity = params.get("granularity")
+        if granularity == "random":
+            granularity = random.choice(params.objects("granularity_list"))
+
+        if granularity:
+            params["granularity"] = int(
+                utils_numeric.normalize_data_size(granularity, "B")
+            )
```

Comment:
```suggestion
        granularities = params.get("granularities")
        if granularities:
            granularities = granularities.split()
            if params.get('choice_granularitiy') == "random":
                granularity = random.choice(granularities)
            else:
                 granularity = granularities[0]
            params["granularity"] = int(
                utils_numeric.normalize_data_size(granularity, "B")
            )
```
Maybe readable since the ` granularity = params.get("granularity") ` is pun from your cfg:
```

        - granularity_512:
            granularity = 512 --> here is a value
        - 2M:
            granularity = 2M
        - random_betwen_512_2M:
            granularity = random --> here is the way to choose
```

Reply:
updated with different style, please double check, thanks